src/nvim/api/vim.c

```diff
@@ -1562,3 +1563,18 @@ Object nvim_get_proc(Integer pid, Error *err)
 #endif
   return rvobj;
 }
+
+/// Give the info of the state of the Caps/Num lock
+///
+/// @return the dictionary holding the data about the locks
```

Comment:
Still not specific. And below line has unwanted space.

Reply:
Ah yes, may be missed this detail, sorry

Reply:
One thing is here unclear, 

```
///
/// @param[out]  dict  Pointer to dictionary where information about modifiers
///                    is to be dumped.
/// @param[out]  err  Location where error message is to be saved, set to NULL
///                   if no error.
///
/// @return true in case of error, false otherwise.
```
The code you provided, there was a space after the params

Reply:
Line below contains function declaration, not documentation. It has space before parenthesis.